plasma_core/utils/exit_priority.py

```diff
@@ -0,0 +1,14 @@
+def parse_exit_priority(priority):
+    return parse_exitable_at(priority), parse_tx_pos(priority), parse_exit_id(priority)
+
+
+def parse_exitable_at(priority):
+    return priority // 2 ** 214  # take 42 most significant bits
```

Comment:
I do agree, but in python it is not that natural, as shifting left will not give the effect (no overflow) and as a result it looks much more difficult than modulo and division.


I have prepared the changes, but leave it to you to decide which version is better 😉

Reply:
Please note that since the fields are not full bytes (42 / 8 = 5 bytes and 2 bits) it would look like 
```
bytes_priority = priority.to_bytes(32, 'big')
return int.from_bytes(bytes_priority[0:6], 'big') >> 2
```

Reply:
I made a commit that takes advantage of bitwise operators, please decide whether it should be merged, or drop it otherwise 😉